qualtran/bloqs/hubbard_model.ipynb

```diff
@@ -31,17 +32,20 @@
     "  - \\frac{u}{4} \\sum_{p,\\sigma} Z_{p,\\sigma} + \\frac{uN}{4}\\mathbb{1}\n",
     "$$\n",
     "\n",
+    "This can be simulated using a qubitization circuit, which consists of PREPARE and SELECT\n",
+    "operations. This module contains `SelectHubbard` and `PrepareHubbard`, with particular\n",
+    "compilation optimizations for the Hubbard model. For more insight into how Select and Prepare\n",
+    "operations can be combined into a quantum walk, please see\n",
+    "[Qubitization Walk Operator](./qubitization_walk_operator.ipynb).\n",
     "\n",
-    "This model consists of a PREPARE and SELECT operation where our selection operation has indices\n",
+    "In these operators, our selection register has indices\n",
```

Comment:
Maybe "with these operators"? In these operators sounds a little off to me...

Reply:
thanks